### PR TITLE
Fix extra new line when adding a use statement

### PR DESCRIPTION
**Pull request by elythyr**
That's an issue I had at work for quite a while and I had it when fixing: https://github.com/phpactor/phpactor/issues/1106

I added some cases to the tests to reproduce the bug and fixed it.
For my understanding the issue came from the `NodeHelper::emptyLinesPrecedingNode()` method.
This method was returning `0` which triggers the add of a new line the following case:
```php
<?php

use SomeClass;

// Comment
$nodeForWhichWeLookForPrecedingEmptyLines = 'something';
```
Resulting in:
```php
<?php

use NewlyAddedUseStatement;
use SomeClass;


// Comment
$nodeForWhichWeLookForPrecedingEmptyLines = 'something';
```

This is because the helper check the number of empty lines between the node (last line) and the comment above.

In order to solve the issue I decided to only do this check when adding imports to a code that has none.
Meaning if we add an import to a code like:
```php
<?php

use SomeClass;
$restOfTheCode = 'test';
```
The result will be:
```php
<?php

use NewlyAddedUseStatement;
use SomeClass;
$restOfTheCode = 'test';
```

I'm personally fine with it, I think the codebuilder should try to follow the PSR as much as possible but it should not change the code it does not generate itself.
If a user has some really strange rules for their code base (we have some really disturbing at work for exmaple...) phpactor should not try to fix it by itself.

Bottom line:
- If there is no use statements, then we do things following the PSR and add a blank line before and after the uses
- If there is already some use statements, then we simply add in the right position inside the exising block without interfering more than that

Is this a policy you can accept or do you prefer for the code builder to also fix the code when it can ?

P.S.: I added the `prefer-stable: true` to the `composer.json`, tell me if you want me to remove it.

### CODE REVIEW

**Comment by dantleech** [comment]
Sounds good to me. I'm not sure why the CI isn't running though ...

**Comment by dantleech** [comment]
But travis was happy anyway (just no update here in GH it seems)
